plugins/inputs/nfsclient/nfsclient_test.go

```diff
@@ -0,0 +1,265 @@
+package nfsclient
+
+import (
+	"bufio"
+	"github.com/influxdata/telegraf/testutil"
+	"strings"
+	"testing"
+)
+
+const mountstatstext = `device rootfs mounted on / with fstype rootfs
```

Comment:
Can we please read this from a file instead of bloating the code!?

Reply:
Moved to a `mountstats` file in the same directory.